### PR TITLE
IN Sage Images

### PR DESCRIPTION
**Pull request by cg2wilson**

### CODE REVIEW

File: source/calculus/source/04-IN/01.ptx [hunk 1]
```diff
@@ -109,9 +109,29 @@
 <activity>
     <introduction> <p>
     The graph of <m>g(t)</m> and the areas <m>A_1, A_2, A_3</m> are given below. </p>
-     <figure xml:id="figure-signed-areas">
-       <image width="100%" source="fig2d.JPG" />
```
[the remark this answers is not in the document]
Comment (cg2wilson) [reply source/calculus/source/04-IN/01.ptx:113]: i think so? i was going to wait to delete the .jpg and .png files until the pull requests were final for each module.

**Comment by jford1906** [comment]
I'm not getting the images to load in the preview. Just a blank box for me. I'm assuming it's on my end, but I want to be sure it's resolved before I approve.

**Comment by siwelwerd** [comment]
> I'm not getting the images to load in the preview. Just a blank box for me. I'm assuming it's on my end, but I want to be sure it's resolved before I approve.

Images don't load in the previews by design to speed up the build process.  General workflow is to build locally to see the images and approve any change.

**Comment by jford1906** [comment]
I'm trying to get it to build with CodeChat in a codespace, but keep getting an error. I'll come back to it later today

**Comment by siwelwerd** [comment]
Hey @jford1906 , remember what I said about "images not loading in the preview"?

That's not true anymore.  See above previews where you can easily preview Cory's work.

**Comment by StevenClontz** [comment]
@jford1906 this is pending your review

**Review by jford1906 — APPROVED** [review]

**Pull request merged**